### PR TITLE
Fix #184 titles on Landing and thank you

### LINKED ISSUE #184: Hard coded page titles on description of Landing and thank you
### Expected behaviour

The landing page and thankyou page titles should be the same as the schema title
### Actual behaviour

They are hardcoded to MCI
### Steps to reproduce the behaviour

Open 1_888.json using /dev and see the landing page title
#### Browser
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] MS Edge
- [ ] Internet Explorer 11
- [ ] Internet Explorer 10
- [ ] Internet Explorer 9
- [ ] Internet Explorer 8
- [ ] Internet Explorer 7
- [ ] Internet Explorer 6
- [ ] Other (please state below)
#### Operating System
- [x] Windows
- [ ] OS X
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by LJBabbage**
### Changes

Fix for #184 
### How to test

see #184 
### Who can test

Anyone but @LJBabbage

### CODE REVIEW

**Comment by collisdigital** [comment]
Add this to unit test?

File: app/templates/landing-page.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h5 class="u-mb-no u-fs-xxs">PERIOD</h5>
     <div>
 
       <div class="header">
-        <h1 class="header__title">Monthly Business Survey - Retail Sales Index</h1>
+        <h1 class="header__title">{{ bar_title }}</h1>
```
Comment (warrenbailey) [inline app/templates/landing-page.html]: I don't think we should be re-using this here, that bar_title variable should only be used for the bar title.

We should actually add a new variable to the data objects passed to these pages

File: app/templates/thank-you.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
   <div class="header u-mb-m u-mb-l@s">
     <h1 class="header__title">Thank You</h1>
-    <div>Monthly Business Survey - Retail Sales Index</div>
+    <div>{{ bar_title }}</div>
```
Comment (warrenbailey) [inline app/templates/thank-you.html]: as above

**Comment by hamishtaplin** [comment]
"Fix for #184" needs to be "Fix #184" 

https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by warrenbailey** [comment]
LGTM 👍

**Pull request merged**